docs/4-explanation/architecture/ci-cd-pipeline-flow.md

```
@@ -1,182 +1,145 @@
 
-# 3. Valide a documentação e anotações atômicas
-make check-docs
+## 6. Gates de Qualidade & Erros Comuns
 
-# 4. Commit e push
-git add .
-git commit -m "feat(scope): descrição da feature"
-git push
-```
+### Gates de Validação
+| Gate | Critério de Aceitação | Recuperação em Caso de Falha |
```

markdownlint-cli2 (0.23.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)